src/components/Search/SearchField/SearchField.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { SearchField } from './SearchField'
+
+describe('SearchField component', () => {
```

Comment:
question: I wonder if it's possible/worth the time to add a test ensuring that the new `inputProps` are passed down as expected? I can go either way on it so I'll leave it up to you :)

Reply:
Thanks for the review! I'll add additional test before I merge this